coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,386 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    Formatting refers to the visual appearance of source code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, strange and inconsistent indentation etc.
+
+        z = 'hello'+'world'
+             def f ( a):
+                pass
+        """
+        example_language = 'python'
+        importance_reason = """
+        A coding style (the of rules or guidelines used when writing the
+        source code) can drastically affect the readability, and
+        maintainability of a program and might as well introduce bugs.
+        """
+        fix_suggestions = """
+        Defining a clearly and thoughtful coding style (based on the available
+        ones given the programming language in use) and strictly respect it or
+        apply it through out the implementation of a project.
+        """
+
+
+@Formatting.subaspect
+class Length:
+    """
+    The `Length` here refers to each line's length and the file length.
```

Comment:
`The Length here refers to...` is unnecessary and confusing. Better just something like `Holds sub-aspects for file and line length`